### DISCUSSION TITLE
[companion] articles found through Google on feed should show in reading history.

### DISCUSSION THREAD

**Discussion by GrahamTheDev** [dailydotdev/daily#611] [Feature Requests]
Now that the companion pops up on all articles found by daily.dev, it would be cool if it added this to your reading history as well, without having to open the article from the feed.

**Comment by AnishDe12020** [comment]
Was about to suggest this lol, this would be an awesome feature!

**Comment by idoshamun** [comment]
Hi @GrahamTheDev,

I will add it to the companion v2 backlog. Thanks :)

**Reply by sashaTen** [reply]
I  will  use   that  feature  .  good  to   have
